INSTALL.md

```diff
@@ -87,6 +96,27 @@ $ kubectl describe rs proxy
 
 If you want to allow images from the Docker Hub, take a look at the `--allow-docker-official` and `--allow-prefixes` arguments. This can be passed to Trow via the `trow.yaml` file.
 
+As an example: editing install/trow.yaml and adding the args:
+```
+containers:
+- name: trow-pod
+  image: containersol/trow:default
+  args: ["-n", "trow:31000 trow.kube-public:31000", "-c", "/certs/domain.crt","--allow-docker-official","--allow-prefixes"]
```

Comment:
`--allow-prefixes` should take an argument. To match my comment above it would be "registry.container-solutions.com/"